pkgs/applications/misc/charm/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildGoModule rec {
   pname = "charm";
-  version = "0.8.5";
+  version = "0.8.6";
 
   src = fetchFromGitHub {
     owner = "charmbracelet";
     repo = "charm";
     rev = "v${version}";
-    sha256 = "0cgl5dpyzc4lciij9q9yghppiclrdnrl1jsbizfgh2an3s18ab8k";
+    sha256 = "0mjq0yy60czsw40h5n515qmi6bbvhrddll4sn5r2q1nf9pvviqr6";
   };
 
   vendorSha256 = "1spzawnk2fslc1m14dp8lx4vpnxwz7xgm1hxbpz4bqlqz1hfd6ax";
```

Comment:
ldflags is missing -s -w. Also according to the doc in language frameworks the double quoting is not required despite it being an array.

Reply:
Thanks! I've applied all changes as requested